### PR TITLE
[TRUNK-5293] Delete statements with empty body

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/TRUNK-5293

### CODE REVIEW

File: api/src/test/java/org/openmrs/test/MigrateDataSet.java [hunk 1]
```diff
@@ -85,28 +85,26 @@ public static void main(String args[]) throws Exception {
 	private static void doMigration(File fileOrDirectory) throws Exception {
 		
 		String filename = fileOrDirectory.getName();
-		
-		if (filename.startsWith(".svn") || filename.endsWith("TestingApplicationContext.xml")) {
-			// skip .svn files
-		} else if (fileOrDirectory.isDirectory()) {
+
+		if (fileOrDirectory.isDirectory()) {
```
Comment (ubap) [inline api/src/test/java/org/openmrs/test/MigrateDataSet.java]: Behaviour is changed here

File: api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java [hunk 2]
```diff
@@ -225,9 +225,7 @@ private void ensurePreferredNameForLocale(Concept concept) {
 			//or the fullySpecifiedName even if not marked preferred
 			ConceptName possiblePreferredName = concept.getPreferredName(locale);
 
-			if (possiblePreferredName != null) {
-				//do nothing yet, but stick around to setLocalePreferred(true)
-			} else if (concept.getFullySpecifiedName(locale) != null) {
+			if (concept.getFullySpecifiedName(locale) != null) {
```
Comment (ubap) [inline api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java]: Behaviour is changed here, is this intentional?

**Comment by BartlomiejRasztabiga** [comment]
@ubap I see, working on it :)

**Comment by BartlomiejRasztabiga** [comment]
@ubap Fixed 😊

**Pull request merged**